README.md

```diff
@@ -10,4 +10,4 @@ ic
 sdk
 ```
 
-To unlock a repository, you should create a [Pull Request](https://github.com/dfinity/repositories-open-to-contributions/compare) where you add the name in alphabetical order.
+To unlock a repository, you should create a [Pull Request](https://github.com/dfinity/repositories-open-to-contributions/compare) where you add the name in alphabetical order. Be sure to commit this directly to the repository, a PR from a forked repository will not work.
```

Comment:
Why it won't work and why can't we make it working?
Also the change is not related to the main fix of this PR I believe.

Reply:
I know - I snuck it in. This is the issue related to not being able to pass in secrets from a forked repo. https://github.com/openmainframeproject/feilong/issues/367

Reply:
I think this is intentional for security reasons.

Reply:
Yes, not sending secrets to pipelines running on forked repositories is intentional. We may still make it working probably as `GITHUB_TOKEN` can be still accessible from PRs on forked repositories, but that should be separate task.
Anyway the comment still looks confusing to me and might be misleading, because "commit this directly to the repository" might be misinterpreted as "commit directly to the master"

Reply:
Sure, the problem is that we can't use GITHUB_TOKEN, but need to use a custom PAT to access org membership. But sure, I'll update the comment. 👍